docs/platforms/native/advanced-usage/backend-tradeoffs/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Backend Tradeoffs
+description: "How to choose the right crash backend in the Native SDK."
+sidebar_order: 1000
+---
+The Native SDK lets users decide at compile-time between three crash backends:
+
+* `crashpad`
+* `breakpad`
+* `inproc`
+
+Currently, `crashpad` is the default on all desktop platforms because it
+
+* has an external `handler` process that allows for external snapshots and sending crash reports immediately (instead of on the next successful start of your application)
+* is the primary target for extension compared to upstream, including
+  * client-side stack traces
```

Comment:
sorry if this is a dumb question, but what does this mean? Is that vs "server-side stack traces"?
Don't we create a minidump with crashpad and hence stack walk on the server?

Reply:
That's not at all a dumb question. We currently have an "experimental" implementation of client-side stack traces for _crashpad_ (which you can enable at compile time) for users who prefer to deploy to environments where they can unwind and symbolicate locally instead of managing their debug symbols at build time.

Minidumps don't preclude adding additional data; in this case, we unwind and symbolicate locally and write a separate stack trace Minidump stream known in the processing pipeline:

https://github.com/getsentry/symbolicator/blob/3d241153e8bbd06bb6bb689f597bdb76c1b7c44e/crates/symbolicator-native/src/symbolication/minidump_stacktraces.rs#L1-L94